.github/workflows/releaseutils.yml

```diff
@@ -0,0 +1,30 @@
+name: Add sbom
+
+on:
+  workflow_dispatch:
+  workflow_run:
+        workflows: [test-latest-release]
+        types: [completed]
+
+permissions: read-all
+
+jobs:
+    add-sbom:
+        name: Add SBOM
+        runs-on: ubuntu-latest
+        if: ${{github.event.workflow_run.conclusion=='success'}}
+        steps:
+            - name: checkout code
+              uses: actions/checkout@v4
+
+            - name: Create SBOM
+              uses: advanced-security/sbom-generator-action@v0.0.1
+              id: sbom
+              env: 
+                  GITHUB_TOKEN: ${{ github.token }}
+              
+            - name: Upload SBOM artefact
+              uses: actions/upload-artifact@v3.1.0

```

Comment:
Do you want an "artifact" or an "asset"? AFAIK, an "artifact" is associated with a workflow run, and "asset" is associated with a release.

Reply:
right, that's why i think the read permission thing might work, for an artefact since its stored with the workflow, it might not required the write permissions, but an asset/package surely does

Reply:
if we need as a release asset, i'll make the required changes

Reply:
I looked at the release assets for Kubestellar, they are in the tar.gz format, what we can do is convert it into the tar.gz format and upload it along with the release as an asset. This would also be helpful later when we decide to sign the artefacts in as mentioned in one of the issues.

Reply:
#2613 seems to be clear that what is needed is an "asset" getting attached to the release.

Reply:
sure